### PR TITLE
Add missing .babelrc and .flowconfig

### PR DESCRIPTION
**Pull request by kalman**

### CODE REVIEW

**Comment by kalman** [comment]
TBR

These used to be in js/noms but that was deleted.

File: samples/js/graphiql/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "license": "Apache-2.0",
   "scripts": {
-    "preinstall": "cp ../../../js/noms/.babelrc ../.eslintrc.js ../../../js/noms/.flowconfig .",
+    "preinstall": "cp ../.babelrc ../.eslintrc.js ../.flowconfig .",
```
Comment (arv) [inline samples/js/graphiql/package.json]: Why not keep these in samples/js/graphiql?

Comment (kalman) [inline samples/js/graphiql/package.json]: Just being consistent with where .flowconfig is.

Comment (arv) [inline samples/js/graphiql/package.json]: Then we should move it. At this point we only have one js sample.

Comment (kalman) [inline samples/js/graphiql/package.json]: Yeah then we wouldn't need the preinstall.

**Pull request merged**